data/pc/1.20.5/protocol.json

```diff
@@ -0,0 +1,7648 @@
+              "type": "varint"
+            },
+            {
+              "name": "reducedDebugInfo",
+              "type": "bool"
+            },
+            {
+              "name": "enableRespawnScreen",
+              "type": "bool"
+            },
+            {
+              "name": "doLimitedCrafting",
+              "type": "bool"
+            },
+            {
+              "name": "worldType",
+              "type": "string"
+            },
+            {
+              "name": "worldName",
+              "type": "string"
+            },
+            {
+              "name": "hashedSeed",
+              "type": "i64"
+            },
+            {
+              "name": "gameMode",
+              "type": "u8"
+            },
+            {
+              "name": "previousGameMode",
+              "type": "i8"
+            },
+            {
+              "name": "isDebug",
+              "type": "bool"
+            },
+            {
+              "name": "isFlat",
+              "type": "bool"
+            },
+            {
+              "name": "death",
+              "type": [
+                "option",
+                [
+                  "container",
+                  [
+                    {
+                      "name": "dimensionName",
+                      "type": "string"
+                    },
+                    {
+                      "name": "location",
+                      "type": "position"
+                    }
+                  ]
+                ]
+              ]
+            },
+            {
+              "name": "portalCooldown",
+              "type": "varint"
+            },
+            {
+              "name": "enforcesSecureChat",
+              "type": "bool"
+            }
+          ]
+        ],
+        "packet_map": [
+          "container",
+          [
+            {
+              "name": "itemDamage",
+              "type": "varint"
+            },
+            {
+              "name": "scale",
+              "type": "i8"
+            },
+            {
+              "name": "locked",
+              "type": "bool"
+            },
+            {
+              "name": "icons",
+              "type": [
+                "option",
+                [
+                  "array",
+                  {
+                    "countType": "varint",
+                    "type": [
+                      "container",
+                      [
+                        {
+                          "name": "type",
+                          "type": "varint"
+                        },
+                        {
+                          "name": "x",
+                          "type": "i8"
+                        },
+                        {
+                          "name": "z",
+                          "type": "i8"
+                        },
+                        {
+                          "name": "direction",
+                          "type": "u8"
+                        },
```

Comment:
these might be double, double, float now 
https://github.com/extremeheat/extracted_minecraft_data/blame/client1.20.5/client/net/minecraft/world/item/component/MapDecorations.java#L27-L30